Scripts/script-FetchFromInstance.yml

```diff
@@ -5,7 +5,11 @@ name: FetchFromInstance
 fromversion: 4.0.0
 script: |-
   import re
+  from distutils.util import strtobool
 
+  incidents_context = []
+  context = {}
+  add_to_context = bool(strtobool(demisto.args().get('add_to_context')))
```

Comment:
Why do you need both strtobool and bool?

Reply:
Because it returns int and I prefer it to be more definitive.